src/pages/home/report/ReportActionCompose.js

```diff
@@ -21,8 +21,8 @@ import ReportTypingIndicator from './ReportTypingIndicator';
 import AttachmentModal from '../../../components/AttachmentModal';
 import compose from '../../../libs/compose';
 import PopoverMenu from '../../../components/PopoverMenu';
+import shouldOverrideRnFocusBehavior from '../../../libs/shouldOverrideRnFocusBehavior';
```

Comment:
I think that ChatGPT had a better name (no offense). Why not `willBlurTextInputOnTapOutside()`? Seem like RN on the native side never blurs and web always does so that's where the difference.

Second thought, is this the right behavior for `react-native-web` to have? I wonder if anyone has ever made this argument:

- `react-native-web` should not blur on outside tap because `react-native` doesn't do that either
- `react-native` should blur on outside tap because that's how all browsers pretty much work by default

Is there an easy answer to that? Maybe it has been brought up before.

Reply:
the 2nd thought is mostly what those other threads i'd started were about and what the follow up issue will cover: standardizing the focus behavior (hopefully).

Reply:
updated the variable name 👍